tests/utils/test_dag_processing.py

```diff
@@ -149,10 +168,11 @@ def test_find_zombies(self):
             ti = TI(task, DEFAULT_DATE, State.RUNNING)
             local_job = LJ(ti)
             local_job.state = State.SHUTDOWN
-            local_job.id = 1
-            ti.job_id = local_job.id
 
             session.add(local_job)
+            session.commit()
+
+            ti.job_id = local_job.id
```

Comment:
This change is _somewhat_ unrelated, I can split it out if people would like.

When re-running individual tests I got a Unique constraint violation on the DB because we aren't using the autoinc PK on the "job" table here.